### ISSUE TITLE
Multimedia Player - Video

### ISSUE THREAD

**Issue by anvistyle** [wet-boew/wet-boew#7972]
Hello 
On this page http://wet-boew.github.io/v4.0-ci/demos/multimedia/multimedia-en.html
You have the error below. 
When I add my video to a page I'm having the same error.
May you please take a look at this problem?
Thanks

wet-boew.min.js:8 Uncaught TypeError: Cannot read property 'defaults' of undefined
    at complete (wet-boew.min.js:8)
    at l (wet-boew.min.js:8)
    at wet-boew.min.js:8
    at h (wet-boew.min.js:8)
    at HTMLScriptElement.n.onreadystatechange.n.onload [as onreadystatechange] (wet-boew.min.js:8)

**Comment by LaurentGoderre** [comment]
In which browser do you get that error?

**Comment by anvistyle** [comment]
Hello
Chrome Version 57
Regards

**Comment by duboisp** [comment]
Here a few additional info about the error

The error occur in the [lightbox plugin at line 251](https://github.com/wet-boew/wet-boew/blob/master/src/plugins/lightbox/lightbox.js#L251) 

Here an extract from line 245 to 255
```javascript
// Load Magnific Popup dependency and bind the init event handler
Modernizr.load( {
	load: "site!deps/jquery.magnific-popup" + wb.getMode() + ".js",
	complete: function() {

		// Set the dependency i18nText only once
		$.extend( true, $.magnificPopup.defaults, i18nText );

		$document.trigger( dependenciesLoadedEvent );
	}
} );
```

**Comment by StdGit** [comment]
@LaurentGoderre @duboisp @CalvinRodo @shawnthompson 
I had similar error in IE 11.0.9600.18860 update 11.049 (KB4052978) 
and firefox 580b15 (64 bits)
Unable to get property 'defaults' of undefined or null reference
At first it works, but after some refresh it crashes at  line 6764

$.extend( true, $.magnificPopup.defaults, i18nText );

After that jquery doesn't run anymore.

**Comment by duboisp** [comment]
This issue seem to have been fixed via PR #8291
